### DISCUSSION TITLE
Disabling Mini.ai does not refresh WhichKey

### DISCUSSION THREAD

**Discussion by thecontinium** [LazyVim/LazyVim#3367] [Q&A]
Hi

When I disable mini.ai either at the global `vim.g.miniai_disable = true `  of buffer `vim.b.miniai_disable = true` level WhichKey still shows the mini.ai settings for text objects.

Is there a way to refresh WhichKey to show the correct details when mini.ai it is disabled ?

Thanks

**Comment by folke** [comment]
nope. will only be hidden when the mini.ai plugin was disabled
